### PR TITLE
PR Chromium 126.2.70 to 127.3.50

### PR DESCRIPTION
**Pull request by SLT-World**
These should be the necessary changes to Chromium 127. Is there anything else missing?

~~I'm not sure why appveyor cannot find 'InitializeCefInternal' function.~~ I just realized that this ChromiumWebBrowser does not use ChromiumWebBrowser.Partial.cs, so I added the InitializeCefInternal function too.

### CODE REVIEW

**Comment by UnknownCoder56** [comment]
cool

File: CefSharp.Wpf.HwndHost/ChromiumWebBrowser.cs [hunk 1]
```diff
@@ -1716,6 +1707,25 @@ public IBrowser GetBrowser()
             return browser;
         }
 
+        private const string CefInitializeFailedErrorMessage = "Cef.Initialize() failed.Check the log file see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file for details.";
```
Comment (amaitland) [inline CefSharp.Wpf.HwndHost/ChromiumWebBrowser.cs:1710]: Please move constants to top of file with the other const, thanks

**Comment by SLT-World** [comment]
Done! I've moved the two constants next to "BrowserNotInitializedExceptionErrorMessage"

**Comment by UnknownCoder56** [comment]
> Done! I've moved the two constants next to "BrowserNotInitializedExceptionErrorMessage"

nice

**Comment by amaitland** [comment]
Thanks 👍 Will try and give it a quick test before publishing nuget packages

**Pull request merged**